template/package.json

```diff
@@ -16,7 +16,7 @@
   "devDependencies": {
     "@babel/core": "^7.12.9",
     "@babel/runtime": "^7.12.5",
-    "@react-native-community/eslint-config": "^2.0.0",
+    "@react-native-community/eslint-config": "^3.0.0",
```

Comment:
@kelset any reason why _not_ to do this? With the current setup, I only locally publish latest versions of packages in the monorepo and `@react-native-community/eslint-config` current major version is 3. Without the bump, the template installation fails since it cannot find a compatible `@react-native-community/eslint-config` version.

Reply:
I think it's fine, I wonder why noone did it before